### ISSUE TITLE
Basic Slurm examples

### ISSUE THREAD

**Issue by davclark** [BIDS-collaborative/dlab-finance#43]
This would be helpful for @juanshishido (as well as other newcomers).

cc @rdhyee @yangraymond @max-eddy

**Comment by rdhyee** [comment]
You mean providing such examples?  I should be able to dig up the work I have done already, refresh my memory on how it worked, and document them as a contribution here.

**Comment by davclark** [comment]
Yeah - I think we already have samples, we just need to point at them, and
maybe also the Savio docs

On Fri, Oct 9, 2015 at 10:06 AM, Raymond Yee <notifications@github.com>
wrote:

> You mean providing such examples? I should be able to dig up the work I
> have done already, refresh my memory on how it worked, and document them as
> a contribution here.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/BIDS-collaborative/dlab-finance/issues/43#issuecomment-146933615>
> .
>



-- 
Dav Clark
Data Scientist
Berkeley D-Lab + BIDS
bead.glass
917-544-8408
